asciidoc.py

```diff
@@ -6410,7 +6406,6 @@ def execute(cmd, opts, args):
     opt_names = [opt[0] for opt in opts]
     if '--doctest' in opt_names:
         # Run module doctests.
-        import doctest
```

Comment:
This should be left here. There's no point promoting it to the global scope and wasting time on it except for the case of us actually running the doctests.

Reply:
Reverted in 49929e9760f944cbc6aeadfdbbf9546a94480b5e; that does make sense, yeah.